<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Middle x/y</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="is_middle_button">
           <property name="styleSheet">
            <string notr="true">font: 18pt &quot;Sans Serif&quot;;</string>
           </property>
           <property name="text">
            <string>Is a Middle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="is_not_middle_button">
           <property name="styleSheet">
            <string notr="true">font: 18pt &quot;Sans Serif&quot;;</string>
           </property>
           <property name="text">
            <string>Is not a Middle</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="status_label">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextBrowser" name="middle_text_viewer">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Sans Serif&quot;;</string>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:9pt;&quot;&gt;this middle reads well&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="note_editor">
         <property name="styleSheet">
          <string notr="true">font: 10pt &quot;Sans Serif&quot;;</string>
         </property>
         <property name="plainText">
          <string>this is a good note</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Which Middle:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="which_middle_spin_box"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="add_middle_button">
           <property name="text">
            <string>Add Middle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="remove_middle_button">
           <property name="text">
            <string>Remove Middle</string>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="reset_entry_button">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Reset Entry</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Subject</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Verb</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Adverb</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="subject_line_edit">
           <property name="text">
            <string>middle</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QSpinBox" name="subject_spin_box"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="verb_line_edit">
           <property name="text">
            <string>reads</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="adverb_line_edit">
           <property name="text">
            <string>well</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="verb_spin_box"/>
         </item>
         <item row="2" column="2">
          <widget class="QSpinBox" name="adverb_spin_box"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="next_button">
           <property name="styleSheet">
            <string notr="true">font: 18pt &quot;Sans Serif&quot;;</string>
           </property>
           <property name="text">
            <string>Next</string>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="previous_button">
           <property name="styleSheet">
            <string notr="true">font: 18pt &quot;Sans Serif&quot;;</string>
           </property>
           <property name="text">
            <string>Previous</string>
           </property>
           <property name="iconSize">
            <size>
             <width>16</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>782</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuControls">
    <property name="title">
     <string>Controls</string>
    </property>
    <addaction name="actionSave_and_Exit"/>
    <addaction name="actionGo_to_next_Unaccounted"/>
   </widget>
   <addaction name="menuControls"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSave_and_Exit">
   <property name="text">
    <string>Save and Exit</string>
   </property>
  </action>
  <action name="actionGo_to_next_Unaccounted">
   <property name="text">
    <string>Go to next Unaccounted</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
